api/pkg/transformer/server/rest/server.go

```diff
@@ -358,3 +358,13 @@ func getHeaders(headers http.Header) map[string]string {
 	}
 	return resultHeaders
 }
+
+// getUrl return url or add default http scheme if scheme is not specified
+func getUrl(rawUrl string) (string, error) {
```

Comment:
why returning err if there is no chance it will throw error?

Reply:
Fixed, I missed removing it after changing the implementation of `getUrl`